### PR TITLE
Added a image preview as small base64 50x50 [NEW]

### PR DESCRIPTION
**Pull request by jorgeluisrezende**
@RocketChat/core 

Open  #5563

Added a image preview as small base64 50x50 pictures embedded on the msg object.

I just made a function, and had call it on the sendFileMessage method.

### CODE REVIEW

File: packages/rocketchat-file-upload/server/methods/sendFileMessage.js [hunk 1]
```diff
@@ -32,42 +32,46 @@ Meteor.methods({
 			title_link_download: true
 		};
 
-		if (/^image\/.+/.test(file.type)) {
-			attachment.image_url = fileUrl;
-			attachment.image_type = file.type;
-			attachment.image_size = file.size;
-			if (file.identify && file.identify.size) {
-				attachment.image_dimensions = file.identify.size;
+		Meteor.wrapAsync(FileUpload.resizeImagePreview(file, Meteor.bindEnvironment(function(r) {
```
Comment (rodrigok) [inline packages/rocketchat-file-upload/server/methods/sendFileMessage.js]: Are you resizing even not image files?

**Comment by rodrigok** [comment]
@jorgeluisrezende Your build is not passing

**Comment by jorgeluisrezende** [comment]
I can fix this, it is a npm package.json problem..

File: packages/rocketchat-file-upload/server/methods/sendFileMessage.js (hunk 1 above)
Comment (jorgeluisrezende) [reply packages/rocketchat-file-upload/server/methods/sendFileMessage.js]: Sorry, I forget to put a test at the function that makes the resize, I will fix this too

Comment (jorgeluisrezende) [reply packages/rocketchat-file-upload/server/methods/sendFileMessage.js]: I added to the file that test, but i'm can not fixing the build error, the log says that are a meteor promise module error.

**Comment by rodrigok** [comment]
@jorgeluisrezende Your JSON at packages/rocketchat-google-vision/.npm/package/npm-shrinkwrap.json is not valid

**Pull request closed without merging**